pkg/commands/compute/language_toolchain.go

```diff
@@ -1,10 +1,562 @@
+// met.
+func (tv ToolchainValidator) compilationVersion(version string) error {
+	fmt.Fprintf(tv.Output, "\nChecking version constraints for '%s'...\n", tv.Compilation)
+
+	v, err := semver.NewVersion(version)
+	if err != nil {
+		err = fmt.Errorf("error parsing version output %s into a semver: %w", version, err)
+		tv.ErrLog.Add(err)
+		return err
+	}
+
+	constraint, ok := tv.Constraints["compilation"]
+	if !ok {
+		err := fmt.Errorf("failed to lookup the compilation constraint: '%s'", tv.Constraints)
+		tv.ErrLog.Add(err)
+		return err
+	}
+
+	c, err := semver.NewConstraint(constraint)
+	if err != nil {
+		err = fmt.Errorf("error parsing compilation constraint %s into a semver: %w", constraint, err)
+		tv.ErrLog.Add(err)
+		return err
+	}
+
+	if !c.Check(v) {
+		err := fsterr.RemediationError{
+			Inner:       fmt.Errorf("version %s didn't meet the constraint %s", version, constraint),
+			Remediation: tv.visitURLRemediation(tv.Compilation, tv.CompilationURL),
+		}
+		tv.ErrLog.Add(err)
+		return err
+	}
+
+	fmt.Fprintf(tv.Output, "Version constraints for '%s' are met\n", tv.Compilation)
+
+	return nil
+}
+
+// buildScript validates the Fastly manifest contains a scripts.build value.
+func (tv ToolchainValidator) buildScript() error {
+	fmt.Fprintf(tv.Output, "\nChecking manifest '%s' contains a build script...\n", manifest.Filename)
+
+	m, err := filepath.Abs(manifest.Filename)
+	if err != nil {
+		tv.ErrLog.Add(err)
+		return fmt.Errorf("failed to construct path to '%s': %w", manifest.Filename, err)
+	}
+
+	// gosec flagged this:
+	// G204 (CWE-78): Subprocess launched with function call as argument or cmd arguments
+	// Disabling as we trust the source of the variable.
+	/* #nosec */
+	bs, err := os.ReadFile(m)
+	if err != nil {
+		err = fmt.Errorf("failed to read file '%s': %w", m, err)
+		tv.ErrLog.Add(err)
+		return err
+	}
+
+	tree, err := toml.LoadBytes(bs)
+	if err != nil {
+		err = fmt.Errorf("failed to parse toml file '%s': %w", m, err)
+		tv.ErrLog.Add(err)
+		return err
+	}
+
+	if v, ok := tree.GetArray("scripts").(*toml.Tree); ok {
+		if script, ok := v.Get("build").(string); ok && script != "" {
+			fmt.Fprintf(tv.Output, "Found [scripts.build] '%s'\n", script)
+			return nil
+		}
+	}
+
+	err = fmt.Errorf("failed to find a [scripts] section with a 'build' property in the '%s' manifest", m)
+	tv.ErrLog.Add(err)
+
+	// We failed to find a [scripts.build] which is required, so we'll set a
+	// default value for the user.
+	tree.Set("scripts.build", tv.DefaultBuildCommand)
+
+	data, err := tree.Marshal()
+	if err != nil {
+		return fmt.Errorf("error updating fastly.toml with a default [scripts.build]: %w", err)
+	}
+
+	err = tv.FastlyManifestFile.Load(data)
+	if err != nil {
+		return fsterr.RemediationError{
+			Inner:       err,
+			Remediation: fmt.Sprintf(fsterr.ComputeBuildRemediation, tv.DefaultBuildCommand),
+		}
+	}
+
+	fmt.Fprintf(tv.Output, "Found default [scripts.build] '%s'\n", tv.DefaultBuildCommand)
```

Comment:
> No build command found.  Patching fastly.toml with the default build command for {language}: {command}

Maybe output this even when not in verbose mode.

Reply:
Also instead of "Building package uisng {language} toolchain" consider:

```
Running [scripts.build]:
> {command}
```

Reply:
@triblondon I've modified the above line 515 as per your suggestion but as for your second comment I've only changed the output to `Running [scripts.build]` and not to include the command output because when not using the `--verbose` flag we need the title to be one line. As demonstrated below...

<img width="979" alt="Screenshot 2022-09-21 at 12 42 58" src="https://user-images.githubusercontent.com/180050/191495322-4ab26077-5f30-4288-bbb1-ed23db664375.png">